apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -256,6 +259,34 @@ export class InstallManager {
 
       ApprovedPackagesChecker.rewriteConfigFiles(this._rushConfiguration);
 
+      // Git hooks are only installed if the repo opts in by including files in /common/git-hooks
+      const hookSource: string = path.join(this._rushConfiguration.commonFolder, 'git-hooks');
+      const hookDestination: string = path.join(this._rushConfiguration.rushJsonFolder, FolderConstants.Git, 'hooks');
+      if (FileSystem.exists(hookSource)) {
+        console.log(os.EOL + colors.bold('Detected /common/git-hooks, clearing and reinstalling Git hooks'));
+        const hookFilenames: Array<string> = FileSystem.readFolder(hookSource);
+
+        if (hookFilenames.length === 0) {
+          console.log(colors.yellow('Warning - You have not added any git hooks yet, skipping for now'));
```

Comment:
I did notice that this second log seems to come up in white instead of yellow. Not sure if this is just a PowerShell-specific thing.

![image](https://user-images.githubusercontent.com/14134000/48102826-b11a9280-e1e1-11e8-8704-136aa713466b.png)

Reply:
Whoops yeah, nevermind, seems to just be the way PowerShell displays it. :)